tasks/fix-cat2.yml

```diff
@@ -811,48 +811,55 @@
       - home
 
 - name: "MEDIUM | RHEL-07-020620 | PATCH | All local interactive user home directories defined in the /etc/passwd file must exist."
-  command: "true"
-  changed_when: no
+  file:
+    path: "{{ item.split(':')[2] }}"
+    state: directory
+  with_items: "{{ rhel7stig_passwd_homedir_check.stdout_lines }}"
```

Comment:
I'd be okay with this if you add "check_mode: yes" and change PATCH to AUDIT in the name.

Reply:
Shouldn't check_mode: yes go in the preliminary task, though? If that's intended as a 'do this even on a dry run' flag.

Reply:
"check_mode: yes" means "make sure this task does not change anything, i.e., 'run it in check mode always'"  That task would always execute as if "--check" had been specified on the command line, even if the play is otherwise not running in check mode.